### PR TITLE
Autofix for jsx-closing-bracket-location.

### PR DESCRIPTION
**Pull request by KevinGrandon**
@yannickcr - I'm fairly new to eslint and the inner workings of this so this could be wrong, or there could be better ways of doing things. Let me know what you think, thanks!

### CODE REVIEW

File: tests/lib/rules/jsx-closing-bracket-location.js [hunk 1]
```diff
@@ -596,6 +656,13 @@ ruleTester.run('jsx-closing-bracket-location', rule, {
       '         />',
       '}'
     ].join('\n'),
+    output: [
+      'var x = function() {',
+      '  return <App',
+      '    foo',
+      '/>',
```
Comment (yannickcr) [inline tests/lib/rules/jsx-closing-bracket-location.js]: I think it should be aligned with the `return` keyword in this case.

**Comment by yannickcr** [comment]
Thanks for this! I think there is a small issue in a test case though.

**Comment by KevinGrandon** [comment]
@yannickcr - Thank you for taking a look. I've fixed the test case and resolved this by walking up nodes until it finds one that it's looking for. Commit is here: https://github.com/yannickcr/eslint-plugin-react/pull/511/commits/0f9ff7f9a5e199618f8373c21a02a9153f3825a7

Please let me know if there's a better way of doing this, or if there's anything else you'd like to see here. Thanks!

**Comment by yannickcr** [comment]
Great! Can you squash your commits? Then I think we'll be good to go :)

**Comment by KevinGrandon** [comment]
This is now squashed, thanks!

**Comment by yannickcr** [comment]
Merged. Thanks!

**Pull request merged**